build.sbt

```diff
@@ -31,24 +51,7 @@ lazy val commonSettings = Seq(
     }
   },
   run / fork := true,
-  developers ++= List(
-    Developer(
-      "bryce-anderson",
-      "Bryce L. Anderson",
-      "bryce.anderson22@gamil.com",
-      url("https://github.com/bryce-anderson")),
-    Developer(
-      "rossabaker",
-      "Ross A. Baker",
-      "ross@rossabaker.com",
-      url("https://github.com/rossabaker")),
-    Developer(
-      "ChristopherDavenport",
-      "Christopher Davenport",
-      "chris@christopherdavenport.tech",
-      url("https://github.com/ChristopherDavenport"))
-  ),
-  startYear := Some(2014)
+  scalafmtConfig := file(".scalafmt.blaze.conf")
```

Comment:
I didn't have the heart to re-format the old or the new, so there are two separate configs now. I leave it to the maintainers to reconcile. I did sacrifice the examples, but anyway.

Reply:
I'd rather reformat the old, because it moves slower than the new, but that can be done in a separate PR.